### PR TITLE
Add irc_msg configuration option for custom announce message.

### PR DESCRIPTION
**Pull request by tumist**

### CODE REVIEW

File: reddit_irc.py [hunk 1]
```diff
@@ -68,7 +80,7 @@ def add(self, server_bot, channel):
     def update(self):
         submissions = []
         try:
-            for submission in self.subreddit.get_new():
+            for submission in self.subreddit.get_new(limit=self.MSG_LIMIT):
```
Comment (bboe) [inline reddit_irc.py]: The limit should not be added here, otherwise new submissions may be missed. The limit is specify how many submissions can be posted to IRC during an update period (30 seconds).

Comment (tumist) [inline reddit_irc.py]: I understand. This change does not belong in this commit.

**Comment by bboe** [comment]
Sorry for taking so long to merge this PR. Thanks!
